src/components/SecondaryPanes/tests/__snapshots__/Expressions.spec.js.snap

```diff
@@ -17,7 +17,6 @@ exports[`Expressions should always have unique keys 1`] = `
         autoExpandDepth={0}
         createObjectClient={[Function]}
         disableWrap={true}
-        focusable={false}
```

Comment:
I'm not sure we want those changes in this file?
I guess it could be because `focusable` defaults to `true`.
If we still want that, maybe we should put `focusable: false` on the `Expression` component.

@darkwing should the Object in the Expressions panel be focusable?

Reply:
AFAIK, this should be focusable or it is simply inaccessible. Changes to tree should make it not annoying to tab through.